examples/modal.py

```diff
@@ -0,0 +1,67 @@
+import discord
+from discord import app_commands, Object, ui
```

Comment:
```suggestion
from discord import app_commands
```

The examples generally don't do imports like these. Might as well be consistent.